src/api/modules/address/address.resolver.ts

```diff
@@ -1,29 +1,20 @@
-import { checkAuthentication } from "../../../core/utils/check-authentication.js";
-import { Context } from "../../context.interface.js";
-import { CreateAddressUseCase } from "../../../domain/use-cases/address/create-address.use-case.js";
-import { GetAddressesByUserIDUseCase } from "../../../domain/use-cases/address/get-addresses-by-user-id.use-case.js";
-import { CustomError } from "../../../domain/errors/index.js";
-import { AddressInput } from "./inputs/index.js";
-import { Address } from "./types/index.js";
+import { checkAuthentication } from "../../../core/utils/check-authentication";
+import { Context } from "../../context.interface";
+import { CreateAddressUseCase } from "../../../domain/use-cases/address/create-address.use-case";
+import { GetAddressesByUserIDUseCase } from "../../../domain/use-cases/address/get-addresses-by-user-id.use-case";
+import { CustomError } from "../../../domain/errors/index";
```

Comment:
```suggestion
import { CustomError } from "../../../domain/errors";
```

aqui e em outros lugares que você utiliza index, não precisa passar o caminho do arquivo index

Reply:
Interessante, não sabia